components/recipesCounter/countAllRecipesOutput.tsx

```diff
@@ -0,0 +1,9 @@
+import React from "react"
+
+export const CountAllRecipesOutput = (props: {value: number}) => {
```

Comment:
```suggestion
export const RecipesCounter = (props: {value: number}) => {
```

plus rename file to recipesCounter.tsx